fcrepo-import/src/main/java/org/fcrepo/importer/ArgParser.java

```diff
@@ -26,13 +27,19 @@
 import org.apache.commons.cli.ParseException;
 
 /**
+ * Command-line arguments parser.
+ *
  * @author awoods
+ * @author escowles
  * @since 2016-08-29
  */
 public class ArgParser implements org.fcrepo.importexport.ArgParser {
 
     final private Options cmdOptions;
 
+    public static final String DEFAULT_RDF_EXT = ".ttl";
```

Comment:
@awoods Yes, but the Importer test needs the ArgParser to parse the options to get the Config.